WebContent/WEB-INF/jsp/collection.jsp

```diff
@@ -43,9 +44,29 @@
 	    var collIndex = document.mainForm.category.selectedIndex;
 		var parentIndex = document.mainForm.parentCollection.selectedIndex;  
 	    if(parentIndex != 0){
-	    	if (category.options[collIndex].value == parentCollection.options[parentIndex].value) {
-	    		alert("Parent collection can't be the same collection. Please select another parent collection.");
+	    	var collTypeValue = document.mainForm.collType.options[collTypeIndex].value;
+	    	var parentColl = document.mainForm.parentCollection.options[parentIndex].text;
+	    	var parentCollType = parentColl.substring(parentColl.lastIndexOf("[")+1, parentColl.lastIndexOf("]"));
+
+			if (collTypeValue == collTypes[2]) { // ProvenanceCollectionPart
+	    		alert("A " + collTypes[2] + " is not allowed to have any parent collections!");
 				return false;
+	    	} else {
+	    		var error = "";
+	    		if (category.options[collIndex].value == parentCollection.options[parentIndex].value) {
+		    		error = "Parent collection can't be the same collection.";
+		   		} else if (collTypeValue == collTypes[1]) { // ProvenanceCollection
+		   			if (parentCollType != collTypes[2])
+	    				error = "A " + collTypes[1] + " can only have a " + collTypes[2] + " parent.";
```

Comment:
I believe this would output: "A ProvenanceCollection can only have a ProvenanceCollectionPart parent." to the user. So I don't think this is the message, or constraint, we want checked here. If the collTypeValue shows the Collection type is a ProvenanceCollection, I'm guessing we either:
a) Don't want a parent at all (An Assembled isn't a true "parent" as far as access control goes) or
b) Only allow an Assembed.

I think this might be switched with the if clause above, since ProvenanceCollectionParts can definitely have parent collections. 

Can you double check please?